src/main/java/com/educare/unitylend/service/BorrowRequestService.java

```diff
@@ -0,0 +1,15 @@
+package com.educare.unitylend.service;
+
+import com.educare.unitylend.Exception.ServiceException;
+import com.educare.unitylend.model.BorrowRequest;
+
+import java.util.List;
+
+public interface BorrowRequestService {
+
+    List<String> getRequestedCommunitiesByUserId(String userId) throws ServiceException;
+
+    boolean raiseBorrowRequestByUserId(String userId, BorrowRequest borrowRequest) throws ServiceException;
+
+    void createBorrowRequest(BorrowRequest borrowRequest) throws ServiceException;
```

Comment:
This function should also return boolean value, Whether the borrow request is created successfully in the DB or not.